web/app/components/thread/getCollapsedComments.js

```diff
@@ -1,5 +1,6 @@
 import { LS_COLLAPSE_KEY } from 'common/constants';
+import { getItem } from 'common/localStorage';
 
-const getCollapsedComments = () => JSON.parse(localStorage.getItem(LS_COLLAPSE_KEY) || '[]');
+const getCollapsedComments = () => JSON.parse(getItem(LS_COLLAPSE_KEY) || '[]');
```

Comment:
It would be better if you imported 'common/localStorage' as `LS`, because in this case it would be `LS.getItem`, and it's more readable than just odd `getItem`.

Reply:
I decided to rather import as `import { getItem as localStorageGetItem } from 'common/localStorage';` as importing * is considered bad practice.